packages/tasks/src/model-libraries.ts

```diff
@@ -666,6 +666,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"models/default.zip"`,
 	},
+	"t5-tts": {
+		prettyLabel: "T5-TTS",
+		repoName: "T5-TTS",
+		repoUrl: "https://github.com/SWivid/F5-TTS",
+		filter: false,
+		countDownloads: `path_extension:"safetensors" OR path_extension:"pt"`,
```

Comment:
Do you know why they went for a solution with both E2_TTS and F5_TTS in the same repo? Doing so doesn't allow for separate counting. If you are in contact with them, would be good to nudge them to split. 

The current download rule looks good though, no matter if the models get split in 2 repos are not.

Reply:
cc @Swivid would you like to have separate download counts for the E2_TTS and F5_TTS checkpoints?

If yes, then it makes sense to split them up into 2 separate model repositories.

Reply:
Would it influences the current spaces using it ?
If not, would like to split it up, otherwise thought it's just OK for having F5 and reproduce E2 there

Reply:
Yes so the current Spaces are assuming both are present in the same model repo, see e.g. https://huggingface.co/spaces/mrfakename/E2-F5-TTS/blob/main/app.py#L53.

So updating that would break existing Spaces

Reply:
Ok, @SWivid would you be up for that?

Reply:
sure, i'll take care of it

Reply:
@NielsRogge sorry for the delay, just got repo split.
will remove E2 weights from F5 repo once PR on mrfakename space merged

Reply:
Thanks! Note that both model repos would require `library_name: f5-tts` for downloads to work.